openfecwebapp/templates/layouts/main.html

```diff
@@ -16,6 +16,7 @@
   {% endif %}
   {% if api_location %}
   <script>
+    ANALYTICS = {{ use_analytics|json }};
```

Comment:
Hm. It seems to work, but in the docs it says `tojson` . http://flask.pocoo.org/docs/0.11/templating/#standard-filters